.github/actions/build/action.yml

```
@@ -15,14 +15,20 @@ inputs:
     default: 'template_debug'
     description: Build type (template_debug or template_release).
   scons-cache:
-    default: .scons-cache/
-    description: Scons cache location.
+    default: '.scons-cache/'
+    description: Scons cache folder name, relative to each scons directory. Must not contain relative path signifiers (. or ..). Must be a transparent path part (empty or 'path/to/folder/', ending in a slash).
```

You forgot that one!

```suggestion
    description: Scons cache folder name, relative to each scons directory. Must not contain relative path signifiers (. or ..). Must be a transparent path part (empty or 'path/to/directory/', ending in a slash).
```